1_processing/3_validate_compendia.py

```diff
@@ -227,21 +225,25 @@
 fig2 += pn.labs(
     x="median expression of PAO1-only genes",
     y="median expression of PA14-only genes",
-    title="MR normalized accessory gene expression in PA14 compendium",
+    title="PA14 compendium accessory gene expression",
     # title="Accessory gene expression in PA14 compendium",
 )
 fig2 += pn.theme_bw()
 fig2 += pn.theme(
     legend_title_align="center",
     plot_background=pn.element_rect(fill="white"),
     legend_key=pn.element_rect(fill="white", colour="white"),
-    legend_title=pn.element_text(family="sans-serif", size=12),
-    legend_text=pn.element_text(family="sans-serif", size=10),
-    plot_title=pn.element_text(family="sans-serif", size=14),
-    axis_text=pn.element_text(family="sans-serif", size=10),
-    axis_title=pn.element_text(family="sans-serif", size=12),
+    legend_title=pn.element_text(family="sans-serif", size=14),
+    legend_text=pn.element_text(family="sans-serif", size=12),
+    plot_title=pn.element_text(family="sans-serif", size=16),
+    axis_text=pn.element_text(family="sans-serif", size=12),
+    axis_title=pn.element_text(family="sans-serif", size=14),
+)
+fig2 += pn.guides(
+    colour=pn.guide_legend(
+        title="SRA strain type", override_aes={"alpha": 1, "size": 3}
```

Comment:
"SRA strain type" mean the strain as annotated on user-submitted SRA metadata? Doesn't necessarily need to be changed, just wanted to check if my interpretation is correct

Reply:
Yes your interpretation is correct here - SRA strain type = strain annotations from SRA metadata